test/Microsoft.ML.Tests/Transformers/FeatureSelectionTests.cs

```diff
@@ -20,7 +20,7 @@ public FeatureSelectionTests(ITestOutputHelper helper)
         {
         }
 
-        [Fact]
+        [Fact(Skip = "FeatureSeclection transform cannot be trained on empty data, schema propagation fails")]
```

Comment:
What is the plan for re-enabling these tests? Typically it is a good practice to log a tracking issue when disabling a test, and putting the issue URL into the Skip argument. That way someone can see why this test was disabled, and who is working on it.